enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/InteractiveDashboard.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ComponentStory } from "@storybook/react";
+
+import { CommonStoryWrapper } from "embedding-sdk/test/common-stories-utils";
+
+import { InteractiveDashboard } from "./InteractiveDashboard";
+
+const DASHBOARD_ID = (window as any).DASHBOARD_ID || "1";
```

Comment:
since dashboard ID could either be a number or a string I think it'd make sense to make the default value a number. What do you think? This applies to all stories.

```suggestion
const DASHBOARD_ID = (window as any).DASHBOARD_ID || 1;
```